index.html

```diff
@@ -26,22 +28,151 @@
+                                        <button type="button" class="btn3 btn-primary btn-lg">Ruby</button>
+                                        <div class ="footer"><button class = "projects btn-primary btn-lg">See project</button></div>
+                                    </div></li>
+                                        <li class="cardlist"> <div id="cwaction1" class="cwaction1">
+                                            <h2 class="card1">Professional Art Printing Data</h2>
+                                            <p class="cardtext">Profesional Art Printing DataA daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                                            <button type="button" class="btn3 btn-primary btn-lg">HTML</button>
+                                                <button type="button" class="btn3 btn-primary btn-lg">BootStrap</button>
+                                                <button type="button" class="btn3 btn-primary btn-lg">Ruby</button>
+                                                <div class ="footer"><button class = "projects btn-primary btn-lg">See project</button></div>
+                                            </div></li>
+                                                <li class="cardlist"> <div id="cwaction1" class="cwaction1">
+                                                    <h2 class="card1">Professional Art Printing Data</h2>
+                                                    <p class="cardtext">Profesional Art Printing DataA daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                                                    <button type="button" class="btn3 btn-primary btn-lg">HTML</button>
+                                                        <button type="button" class="btn3 btn-primary btn-lg">BootStrap</button>
+                                                        <button type="button" class="btn3 btn-primary btn-lg">Ruby</button>
+                                                        <div class ="footer"><button class = "projects btn-primary btn-lg">See project</button></div>
+                                                    </div></li>
+        </ul>
+    </section>
+    <section id="about-section" class="about-section">
+        <div class="about-introduction-content">
+          <h1 class="about-heading">About me</h1>
+          <p>
+            Hey there! I can help you build a product, feature or website. Look
+            through some of my work and experience! If you like what you see and
+            have a project you need coded, don't hesitate to contact me.
+          </p>
+          <button class="my-resume-button" type="button">Get My Resume</button>
+        </div>
+        <div class="background-graphic"></div>
```

Comment:
- [ ] The background image that you are applying here is not working. kindly consider fixing that and consider applying background image to this section as in the figma design